content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -4,15 +4,18 @@ This tutorial guides you through the basics of building an XRP Ledger-connected
 
 The scripts and configuration files used in this guide are [available in the Ripple Dev Portal GitHub Repository](https://github.com/ripple/ripple-dev-portal/tree/master/content/_code-samples/rippleapi_quickstart).
 
-# Environment Setup
+
+
+## Set Up Your Environment
```

Comment:
I'm inclined not to change these anchors unless there's a clear improvement, because it breaks old links to them, including ones on external sites. IMO this difference isn't quite significant enough to justify changing.